src/logs/metadata_engine/metadata_engine.py

```diff
@@ -194,13 +210,16 @@ def _create_attributes(attributes_json: List[Dict]) -> List[Attribute]:
 
     for source_json in attributes_json:
         key = source_json.get("key", None)
+        priority = source_json.get("priority", None)
         pattern = source_json.get("pattern", None)
 
         if key and pattern:
-            result.append(Attribute(key, pattern))
+            result.append(Attribute(key, priority, pattern))
         else:
             logging.warning(f"Encountered invalid rule attribute with missing parameter, parameters were: key = {key}, pattern = {pattern}")
 
+    # attributes without priority are executed last
+    result.sort(key= lambda attribute: attribute.priority if attribute.priority is not None else inf)
```

Comment:
You know what. I think we are sorting entries in static json lots of times. Probably there is a readability reason for some specific ordering but maybe we should consider sorting it once and adjusting static json so we don't have to sort it in runtime anymore

Reply:
Ok, my mistake, this load&sort is only executed once per lambda run so not an issue 👍